analytics_data_api/utils.py

```diff
@@ -98,3 +53,10 @@ def dictfetchall(cursor):
         dict(zip([col[0] for col in desc], row))
         for row in cursor.fetchall()
     ]
+
+
+def class_for_name(path):
+    """ Returns the class given the full path. """
```

Comment:
Could you clarify what `path` is expected to be? For example, it seems like you're expecting a fully-qualified class name. I usually associate 'path' with a slash-separated directory or URL structure.